.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
do we need this file?

Reply:
Not exactly sure who added this @gachouchani1999 @aelesbao but I assume it's related to running the CLI from vscode